about/index.html

```diff
@@ -14,43 +14,43 @@
 
     <title>Who Travels - About</title>
 </head>
-<body>
-    <header>
-        <!-- Header -->
-        <section>
-            <div class="nav-container">
-                <a href="../index.html"><img src="../assets/img/logos/logo-icon/logo-icon-white.svg" alt="Who Travels Logo" class="logo"></a>
-                <nav class="nav-menu">
-                    <ul class="menu">
-                        <li class="nav-li"><a href="../book/" class="top-nav" id="book">Book Trip</a></li>
+
+   <header>
+       <!-- Header -->
+       <section>
+           <div class="nav-container">
+               <a href="/index.html"><img src="/assets/img/logos/logo-icon/logo-icon-white.svg" alt="Who Travels Logo" class="logo"></a>
```

Comment:
about nav - index and logo missing link fix update

Reply:
@lala-laura just updated, also fixed some links in destinations and contact